libs/shared-atomic-design-components/src/lib/atoms/AppNotification.tsx

```diff
@@ -0,0 +1,70 @@
+import { Slide, Snackbar, SnackbarContent } from '@mui/material'
+import { useCallback, useEffect, useState } from 'react'
+import { Subject } from 'rxjs'
+import { Optional } from 'utility-types'
+
+const subjectShowNotification = new Subject<SubjectShowNotification>()
+
+export interface AppNotificationOptions {
+  autoHideDuration: number | undefined
+}
+interface SubjectShowNotification extends AppNotificationOptions {
+  message: string | null
+}
+type AppNotificationState = SubjectShowNotification | null
+
+export const showNotification = (
+  message: string,
+  options?: Optional<AppNotificationOptions>
+) => {
+  // console.log('show', message)
+  const finalOptions: AppNotificationOptions = {
+    autoHideDuration: 3000,
+    ...options,
+  }
+  subjectShowNotification.next({ ...finalOptions, message })
+}
+
+export function AppNotification() {
+  const [state, setState] = useState<AppNotificationState>(null)
+  const [key, setKey] = useState(0)
+
+  const show = useCallback((options: SubjectShowNotification) => {
+    console.log('yo')
```

Comment:
As @katamatata mentions below, let's remove this "yo" 🪀

Reply:
:D